bigtop-packages/src/rpm/madlib-adb6/SPECS/madlib-adb6.spec

```diff
@@ -0,0 +1,49 @@
+%define debug_package %{nil}
+
+Name: madlib-adb6
+Version: %{madlib_adb6_version}
+Release: %{madlib_adb6_release}
+Summary: Apache MADlib for adb 6 is an Open-Source Library for Scalable in-Database Analytics
+Group: Development/Libraries
+
+Buildroot: %(mktemp -ud %{_tmppath}/%{madlib_adb6_name}-%{version}-%{release}-XXXXXX)
+License: ASL 2.0
```

Comment:
Wrong license

Reply:
https://github.com/apache/madlib/blob/master/LICENSE

Reply:
The same license was in original rpm